src/ResourceManager/EventHub/Commands.EventHub/help/New-AzureRmEventHub.md

```diff
@@ -12,18 +12,17 @@ Creates a new Event Hub.
 
 ## SYNTAX
 
-### EventhubInputObjectSet
+### EventhubPropertiesSet (Default)
 ```
 New-AzureRmEventHub [-ResourceGroupName] <String> [-Namespace] <String> [-Name] <String>
- [-InputObject <PSEventHubAttributes>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+ [-MessageRetentionInDays <Int64>] [-PartitionCount <Int64>] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
-### EventhubPropertiesSet
+### EventhubInputObjectSet
 ```
-New-AzureRmEventHub [-ResourceGroupName] <String> [-Namespace] <String> [-Name] <String>
- [-MessageRetentionInDays <Int64>] [-PartitionCount <Int64>] [-DefaultProfile <IAzureContextContainer>]
- [-WhatIf] [-Confirm] [<CommonParameters>]
+New-AzureRmEventHub [-InputObject <PSEventHubAttributes>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
```

Comment:
Looks like this parameter set doesn't have -ResourceGroupName, -Namespace, or -Name - please add them back.

Reply:
resolved